docs/reference/content/reference/cursors.md

```diff
@@ -0,0 +1,159 @@
++++
```

Comment:
I have no idea what this does, I just included it b/c the other tutorials include it